python/lsst/verify/tasks/commonMetrics.py

```diff
@@ -145,3 +150,131 @@ def makeMeasurement(self, timings):
     @classmethod
     def getOutputMetricName(cls, config):
         return Name(config.metric)
+
+
+class MemoryMetricConfig(MetadataMetricTask.ConfigClass):
+    """Information that distinguishes one memory metric from another.
+    """
+    target = pexConfig.Field(
+        dtype=str,
+        doc="The method to profile, optionally prefixed by one or more tasks "
+            "in the format of `lsst.pipe.base.Task.getFullMetadata()`.")
+    metric = pexConfig.Field(
+        dtype=str,
+        doc="The fully qualified name of the metric to store the "
+            "memory information.")
```

Comment:
They can't go in `MetadataMetricConfig` because they're specific to `@timeMethod`-based metrics. On the other hand, an (internal) `TimeMethodMetricConfig` might be useful.